examples/gulp/build/index.html

```diff
@@ -0,0 +1,10 @@
+<html>
```

Comment:
That's not what all the other examples did. My guess is it's to make it possible for developers to see the result of the build here on GitHub without having to run the example for themselves.